index.js

```diff
@@ -174,12 +179,14 @@ async function main() {
 		const keymanagerUrls = params.keymanagerUrls.split(",").map(s => s.trim());
 		for (let i = 0; i < keymanagerUrls.length; i++) {
 			const keymanagerUrl = keymanagerUrls[i];
+			const token = fs.readFileSync(params.keymanagerTokenFile, 'utf-8');
```

Comment:
Make sure the user is getting a friendly error message when an invalid toke file is provided or if there are permission issues with accessing the file, etc.

Reply:
The changes I made are in the validation of a variable to have the following scenarios:
* If the variable is not set, it will give the following error: "KEYMANAGER_TOKEN_FILE is not defined.".
* If the file exists but is empty, it will give the following error: "Token file is empty."
* If the file does not exist or cannot be accessed, it will give the following error: "The token file cannot be found or does not have access rights. Please enter a valid file path."